### ISSUE TITLE
Config is not persistent in Docker

### ISSUE THREAD

**Issue by bishalkc** [datosgobar/portal-andino-theme#10]
When the theme changes are made in docker environment, say add About/Organization, Portal Name, twitter etc, after the container is restarted all the customizations are lost? Thoughs?

![screen shot 2016-12-18 at 11 12 15 am](https://cloud.githubusercontent.com/assets/5494719/21294774/10c0e1dc-c513-11e6-9969-1cd71668f258.png)

**Comment by JoseSalgado1024** [comment]
¡Hi, @bishalkc!

First of all, thanks for openning an issue. We really appreciate having feedback from the community.

There is a little misunderstood. This repository hosts the code of the ckan-extension of the project and is independent from the final inside-docker instance. [This repository](https://github.com/datosgobar/portal-andino) has the docker files and relevant documentation concerning the containers.

Having said that, we forgot to mention in the setup readme that there are two different commands to run the app. One meant for the first run and another for the subsecuent runnings, and if you always use the first command then you end up losing all your previous configuration. 

The first-time command is `docker run -d --link pg-ckan:db --link solr-ckan:solr -p 80:80 -p 8800:8800 --name app-ckan datosgobar/app-ckan:latest`. This command setups the necessary links between the containers and starts a clean instance of the application.

To stop containers you can use `docker stop pg-ckan solr-ckan app-ckan` and the next time you want to run the app, the exact command is `docker start pg-ckan solr-ckan app-ckan`. 

Tell us if this solves your problem. 

Have a nice week,
The Argentine data team

**Comment by bishalkc** [comment]
Jose,

Thanks for the quick response and sorry for misleading title. I was missing settings.json as part of deployment, and now everything looks good. I still have to work on creating different views (map and graphs). Also I have translated theme in English (still needs some work) but initial translation has been complete. I forked the repo and will be pushing to [https://github.com/bishalkc/portal-andino-theme](url) shortly. 

Feature requests for improvising theme.

- [ ] CSS styling (add few more color schemes to the theme)

- [ ] Multiple language options (defaults to one language and option to configure other, may be link next to About to select **English** )
